helm/cluster-vsphere/templates/cloud-provider-vsphere-helmrelease.yaml

```diff
@@ -52,6 +52,12 @@ spec:
         thumbprint: "{{ .Values.vcenter.thumbprint }}"
     kube-vip-cloud-provider:
       cidrGlobal: '{{ join "," .Values.connectivity.network.loadBalancers.cidrBlocks }}'
+      image:
+        repository: docker.io/giantswarm/kube-vip-cloud-provider
+        tag: v0.0.4
     kube-vip:
```

Comment:
let's keep the app repo up-to-date
https://github.com/giantswarm/cloud-provider-vsphere-app/pull/68/files